genestack-application-manager

```diff
@@ -438,6 +461,25 @@ def show_info(files, vendor_only, with_filename, no_filename):
         first_file = False
 
 
+def prompt_removing_stable_version(application, apps_ids, version):
+    apps = []
+    for app_id in apps_ids:
+            stable_versions = application.invoke('getStableVersions', app_id)
+            if stable_versions.get('SYSTEM') == version:
+                apps.append(app_id)
+
+    if len(apps) == 0:
+        return True
+
+    message = ('You are going to make unstable following system stable applications with version "%s": \n' % version) +\
+        '\n'.join(apps) + "\nDo you want to continue? [Y/n]"
```

Comment:
It is common practice to trigger capital letter by pressing enter. In our case it will be better to make them both in lowercase.